ragna/deploy/_ui/modal_configuration.py

```diff
@@ -297,41 +315,122 @@ def toggle_card(event):
                 toggle_button.name = toggle_button.name.replace("▶", "▼")
 
         toggle_button = pn.widgets.Button(
-            name="Advanced Configurations   ▶",
+            name="Advanced Configuration of Assistants and Source Storages   ▶",
             button_type="light",
             css_classes=["modal_configuration_toggle_button"],
         )
-
         toggle_button.on_click(toggle_card)
 
-        toggle_button.js_on_click(
-            args={"card": card},
-            code=js.JS_TOGGLE_CARD,
+        return pn.Column(toggle_button, card)
+
+    def advanced_config_corpus(self):
```

Comment:
New function for `corpus` tab (but very similar to the `upload` tab function, minus a few options).